packages/web-components/fast-components/src/styles/recipes.ts

```diff
@@ -238,6 +238,7 @@ export const neutralFillStealthRestBehavior = cssCustomPropertyBehaviorFactory(
     x => neutralFillStealth(x).rest,
     FASTDesignSystemProvider.findProvider
 );
+
```

Comment:
Looks like blank lines were added but this file doesn't have them anywhere else.

Reply:
Maybe prettier? not sure, since I took stuff out I should just remove this from the PR